cypress/support/commands.js

```diff
@@ -23,3 +23,23 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+import adminUser from "../fixtures/adminUser.json";
+import studentUser from "../fixtures/studentUser.json";
+import guestUser from "../fixtures/guestUser.json";
+
+Cypress.Commands.add("loginAsAdmin", () =>
+  cy.setCookie("AUTH", JSON.stringify(adminUser))
+);
+Cypress.Commands.add("loginAsStudent", () =>
+  cy.setCookie("AUTH", JSON.stringify(studentUser))
+);
+Cypress.Commands.add("loginAsGuest", () =>
+  cy.setCookie("AUTH", JSON.stringify(guestUser))
+);
+
+Cypress.Commands.add("prepareDatabase", () => {
+  cy.visit("http://localhost:3000/testhooks");
+  cy.get("button").contains("Prepare Database").click();
+  cy.get("span").contains("Database has been reset; ready to run tests.");
```

Comment:
Do we really want to implement this by creating a frontend page for this? If all we need to do is hit a backend endpoint, we should be able to do this with [`cy.request`](https://docs.cypress.io/api/commands/request.html)

Reply:
Good point.    Can we let this go for now, and add a refactoring story to the backlog?